framework/core/src/Api/Controller/DeleteAccessTokenController.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+/*
+ * This file is part of Flarum.
+ *
+ * For detailed copyright and license information, please view the
+ * LICENSE file that was distributed with this source code.
+ */
+
+namespace Flarum\Api\Controller;
+
+use Flarum\Http\AccessToken;
+use Flarum\Http\RequestUtil;
+use Flarum\User\Exception\PermissionDeniedException;
+use Illuminate\Contracts\Session\Session;
+use Illuminate\Database\Eloquent\ModelNotFoundException;
+use Illuminate\Support\Arr;
+use Laminas\Diactoros\Response\EmptyResponse;
+use Psr\Http\Message\ServerRequestInterface;
+
+class DeleteAccessTokenController extends AbstractDeleteController
+{
+    /**
+     * {@inheritdoc}
+     */
+    protected function delete(ServerRequestInterface $request)
```

Comment:
Is there any reason we don't use the Command and CommandHandler system for this like we do with most other models?

Reply:
it's not a must, and this is due to the fact the access token was created on a simple controller in the first place. Wecan improve this in the future if we wish.